src/lib.rs

```diff
@@ -21,7 +21,7 @@
 //
 // This means that missing docs will still fail CI, but means we can use
 // README.md as the crate documentation.
-#![cfg_attr(feature = "nightly", deny(missing_docs))]
+//#![cfg_attr(feature = "nightly", deny(missing_docs))]
```

Comment:
Why did this get commented out?

Reply:
That's my mistake, it was for refactoring.

Reply:
This is fixed in a fixup commit now.